main/src/cgeo/geocaching/settings/Settings.java

```diff
@@ -368,7 +368,7 @@ private static void migrateSettings() {
             legacyGlobalSettings.put(GeocacheFilter.QuickFilter.FOUND, !legacyExcludeFound);
             legacyGlobalSettings.put(GeocacheFilter.QuickFilter.DISABLED, !getBooleanDirect("excludedisabled", false));
             legacyGlobalSettings.put(GeocacheFilter.QuickFilter.ARCHIVED, !getBooleanDirect("excludearchived", false));
-            legacyGlobalSettings.put(GeocacheFilter.QuickFilter.HAS_OFFLINE_LOG, !getBooleanDirect("excludeofflinelog", false));
+            legacyGlobalSettings.put(GeocacheFilter.QuickFilter.HAS_OFFLINE_FOUND_LOG, !getBooleanDirect("excludeofflinelog", false));
```

Comment:
This is the migration routine for legacy c:geo versions before filtering was introduced. Your change would migrate the legacy global setting to filter for offline logs to the new "found offline log" instead of the old "offline log". Is this intended?

Reply:
The new quick settings will no longer support `HAS_OFFLINE_LOG`. So we can either migrate the filter setting to `HAS_OFFLINE_FOUND_LOG` or drop it (it was only used in early beta). 

I can remove it, I don't have a real preference...

Reply:
I removed it now. It's only about early beta users anyway... (they hopefully know how to open the quicksettings ;-))